custom_mxnet.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Will install a custom mxnet version on py3.5env virtualenv
+# Tested on Ubuntu deep learning AMI https://aws.amazon.com/marketplace/pp/B06VSPXKDX
+set -e
+set -x
+VENV=py3.5env
```

Comment:
You can rename it as custom_mxnet_py3.sh
However, a python2/3 compatible script is recommended.